.github/workflows/CI.yaml

```diff
@@ -326,7 +326,47 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           path: artifact
+  
+  build-win:
+    needs: spec_version
+    runs-on: windows-latest
```

Comment:
I guess `windows-latest` host will be too slow. Nightly build is about 50 mins: https://github.com/gear-tech/builds/runs/7014779874?check_suite_focus=true

Reply:
We build in debug mode so I expect it will be faster